packages/paste-website/src/components/CookieConsent.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+
+export const CookieConsent: React.FC<{onAccept: () => void; onReject: () => void}> = ({onAccept, onReject}) => {
+  return (
+    <Box
+      zIndex="zIndex90"
+      position="fixed"
+      bottom="0"
+      width="100%"
+      backgroundColor="colorBackgroundBody"
+      borderTopStyle="solid"
+      borderTopWidth="borderWidth10"
+      borderTopColor="colorBorderWeaker"
+      paddingY="space60"
+      paddingX="space40"
+      display="flex"
+      justifyContent="space-around"
+    >
+      <Box display="flex" alignItems="center" columnGap="space40" maxWidth="1200px">
+        To enhance our site, we utilize tools that establish cookies. The data gathered by these tools is anonymized. If
```

Comment:
I liked the old version more, made it more human.

```suggestion
        To help us improve this site we use tools that set cookies. The data gathered by these tools is anonymized. If you reject the use of cookies, no analytics service will be initiated.
```

Reply:
+1 but add a comma: "...improve this site, we use tools"